torch/hub.py

```diff
@@ -423,7 +427,7 @@ def _download_url_to_file(url, dst, hash_prefix=None, progress=True):
             _download_url_to_file will be removed in after 1.3 release')
     download_url_to_file(url, dst, hash_prefix, progress)
 
-def load_state_dict_from_url(url, model_dir=None, map_location=None, progress=True, check_hash=False):
+def load_state_dict_from_url(url, model_dir=None, map_location=None, progress=True, check_hash=False, **headers):
```

Comment:
My first idea to specify this `**kwargs` might not work as planned since `hub.load()` already has such an argument, but should also have the option the add the request headers. Maybe passing `headers=None` is a better idea?

Reply:
Hmmm does `headers={}` work?

Reply:
Not sure if I got you right, but I would avoid using a mutable as a default argument.